From a2a70c7376f6cf3d6161441d2a59e93f83ba21a5 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 6 Feb 2018 11:10:53 +0100
Subject: [PATCH] network: Really fix clone MAC support

This isn't a container, but a text entry. Bug introduced in commit
6b043fd.
---
 panels/network/connection-editor/ce-page-ethernet.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/panels/network/connection-editor/ce-page-ethernet.c b/panels/network/connection-editor/ce-page-ethernet.c
index dbca6b934..67c4478eb 100644
--- a/panels/network/connection-editor/ce-page-ethernet.c
+++ b/panels/network/connection-editor/ce-page-ethernet.c
@@ -96,11 +96,8 @@ ui_to_setting (CEPageEthernet *page)
                 device_mac = ce_page_trim_address (text);
         }
 
-        entry = gtk_bin_get_child (GTK_BIN (page->cloned_mac));
-        if (entry) {
-                text = gtk_entry_get_text (GTK_ENTRY (entry));
-                cloned_mac = ce_page_trim_address (text);
-        }
+        text = gtk_entry_get_text (GTK_ENTRY (entry));
+        cloned_mac = ce_page_trim_address (text);
 
         g_object_set (page->setting_wired,
                       NM_SETTING_WIRED_MAC_ADDRESS, device_mac,
-- 
2.14.3

